content/cohorts/ds-ai/ottaa-project.en.adoc

```diff
@@ -0,0 +1,34 @@
+---
+title: OTTAA Project
+weight: 50
+categories: "team"
+tags: ["team"]
+type: profiles
+
+description: An Alternative Augmentative Communication (AAC) system, intended for people with speech disabilities.
+country: Chile
+date: 2020-09-25
+github: https://github.com/OTTAA-Project
+linkedin: https://www.linkedin.com/company/ottaa-project/
+twitter: https://twitter.com/OTTAAProject
+website: https://www.ottaaproject.com/
+community: ""
+logo: /inventory/cohorts/logo-ottaa-project.png
+sdgs:
+    - 4
```

Comment:
-SDG 3 , 4 , 10

Reply:
Resolved in commit 3b26cb44151b7969e43c2a1fafa916faeede880a.